bigframes/core/compile/aggregate_compiler.py

```diff
@@ -382,6 +418,19 @@ def _(
     )
 
 
+@compile_binary_agg.register
+def _(
```

Comment:
Not a change introduced in this PR, but it's a bit odd to see functions with name `_` outside of a closure function. Even though the name isn't really relevant, as we're relying on type-based dispatching, I'd expect more than a 1-character name.

https://google.github.io/styleguide/pyguide.html#3161-names-to-avoid

Reply:
The code examples for @singledispatch in https://docs.python.org/3/library/functools.html use the `_` naming for overrides. Also the linter wants unique name for each implementation if it is not `_`. I can change though if this goes against our style guidelines.